Makefile

```diff
@@ -94,6 +94,10 @@ GO_BUILD_FLAGS :=-tags 'include_gcs include_oss containers_image_openpgp gssapi
 GO_TEST_FLAGS=$(GO_BUILD_FLAGS)
 GO_TEST_PACKAGES=./cmd/... ./pkg/...
 
+# Shellcheck files to ignore, this is done to avoid commiting any changes to those files that require
+# upstream change first.
+SHELLCHECK_IGNORE=configure-ovs.*.sh
+
```

Comment:
Let's add this setting in the block of shellcheck invocation, rather than global?